src/test/java/org/jabref/logic/importer/fetcher/SpringerFetcherTest.java

```diff
@@ -78,4 +82,110 @@ void testSpringerJSONToBibtex() {
+
+        List<BibEntry> result = fetcher.performSearch("name:\"Steinmacher, Igor\" AND name:\"Gerosa, Marco\" AND name:\"Conte, Tayana U.\"");
+
+        Assertions.assertEquals(Collections.singletonList(expected), result);
+    }
+
+    @Test
+    @Override
+    public void supportsYearSearch() throws Exception {
+        List<BibEntry> result = fetcher.performSearch("name:\"Steinmacher, Igor\" AND name:\"Gerosa, Marco\" AND year:2014");
+
+        // There are 3 papers published by Igor Steinmacher and Marco Gerosa in 2014.
+        assertEquals(3, result.size());
+        long publicationsIn2014 = result.stream()
+                                        .map(bibEntry -> bibEntry.getField(StandardField.YEAR))
+                                        .filter(Optional::isPresent)
+                                        .map(Optional::get)
+                                        .filter(s -> s.equals("2014"))
+                                        .count();
+        assertEquals(3, publicationsIn2014);
+    }
+
+    @Test
+    @Disabled("Is not natively supported by the API, can be emulated by multiple single year searches.")
+    @Override
+    public void supportsYearRangeSearch() throws Exception {
+    }
+
+    @Test
+    @Override
+    public void supportsJournalSearch() throws Exception {
+
+        List<BibEntry> result = fetcher.performSearch("journalid:392");
+        List<String> resultEntriesJournals = result.stream()
+                                                   .map(bibEntry -> bibEntry.getField(StandardField.JOURNAL))
+                                                   .filter(Optional::isPresent)
+                                                   .map(Optional::get)
+                                                   .collect(Collectors.toList());
+
+        // Ensure no entries without a journal field were returned
+        assertEquals(result.size(), resultEntriesJournals.size());
+        resultEntriesJournals.forEach(journal -> journal.equals("Clinical Research in Cardiology"));
```

Comment:
I think an assertion is missing here, moreover it seems `Assert.equals(Collection.singleton("Clinical..."), result)` is what were looking for.

Reply:
Thanks, I added the missing assert and modified the test.